ui/components/ui/box/box.d.ts

```diff
@@ -314,7 +314,7 @@ export interface BoxProps extends React.HTMLAttributes<HTMLElement> {
    * Use BackgroundColor enum from '../../../helpers/constants/design-system';
    * Accepts responsive props in the form of an array.
    */
-  backgroundColor?: BackgroundColor | BackgroundColorArray;
+  backgroundColor?: BackgroundColor | BackgroundColorArray | Color;
```

Comment:
Edit this can be removed. TS catches another instance of an incorrectly used color nice!
```suggestion
  backgroundColor?: BackgroundColor | BackgroundColorArray;
```

Reply:
Okay!